addons/test_mail/tests/test_message_management.py

```diff
@@ -43,7 +43,9 @@ def test_mail_resend_workflow(self):
 
         # three more failure sent on bus, one for each mail in failure and one for resend
         self._reset_bus()
-        with self.mock_mail_gateway(), self.assertBus([(self.cr.dbname, 'res.partner', self.partner_admin.id)] * 3):
+        data = list()
+        [data.append((self.cr.dbname, 'res.partner', id)) for id in (self.partner_admin.id, self.env.user.partner_id.id)]
```

Comment:
You could build the list directly instead of starting it empty and then appending to it.
But in this case I think it's more readable to not even use a loop.

Also use a more explicit variable name such as `expected_bus_notifications`
```suggestion
        expected_bus_notifications = [
            (self.cr.dbname, 'res.partner', self.partner_admin.id),
            (self.cr.dbname, 'res.partner', self.env.user.partner_id.id),
        ]
```